Project 4/Ali_Project4

```diff
@@ -0,0 +1,101 @@
+PREFIX ex: <http://example.org/ns/>
+PREFIX wiki: <http://dbpedia.org/resource/>
+PREFIX schema: <http://schema.org/>
+PREFIX rdf: <http://w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX owl: <http://w3.org/2002/07/owl#>
+PREFIX rdfs: <http://w3.org/2000/01/rdf-schema#>
+
+
+
+Ali's Kata 8 question: Find the area code used for phone lines in Seattle.
+
+SELECT ?o
+WHERE {dbr:Seattle dbo:areaCode ?o}
+
+--------------------------
+Ali’s Kata 7 question: SELECT queries are used for selecting bindings for variables, but you can also ask YES/NO questions that can be used to decide whether to include a specific section of the reporting software, or support software to make recommendations.
+To do so, we use the ASK keyword. For example, for the question “Did Elton John have an occupation?” we could ask
+
+ASK WHERE { dbr:Elton_John dbo:occupation ?any }  
+
+The answer would be True (yes). Your assignment is to ask a similar question about Aristotle by using FILTER.
+You must also have your answer as False (no).
+
+Solution:
+ASK WHERE { FILTER NOT EXISTS { dbr:Elton_John dbo:occupation ?any }}
```

Comment:
Why is Elton John in the solution again?

Reply:
Second that.